### PR TITLE
Adds SubQuery to Integration section

### PR DESCRIPTION
**Pull request by seandotau**
This PR adds information about how to index blockchain data with SubQuery to help Agoric developers source blockchain data to power their dApps.

### CODE REVIEW

File: main/guides/subquery-indexing.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+You can also follow along this [step by step guide](https://academy.subquery.network/quickstart/quickstart_chains/cosmos-agoric.html) to get familiar with SubQuery or check out the [Agoric x SubQuery workshop](https://www.youtube.com/watch?v=QC5wQOcWynU) to see an actual demo.
+
+## Running and Hosting your Agoric SubQuery APIs
+SubQuery is open-source, meaning you have the freedom to run it in the following three ways:
+
+* Locally on your own computer, or a cloud provider of your choosing. View the instructions on how to run SubQuery locally [here](https://academy.subquery.network/run_publish/run.html).
+
+* You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/login), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```
Comment (michaelfig) [inline main/guides/subquery-indexing.md:21]: I'm adverse to strong claims when writing a guide that we expect to provide as part of Agoric documentation.   Can this paragraph be simplified while still preserving the intent of promoting the managed service?

```suggestion
* You can publish it to SubQuery's [Managed Service](https://managedservice.subquery.network/login), where the SubQuery corporation will host your SubQuery project.  There is also a free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```

File: main/guides/subquery-indexing.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* You can publish it to the decentralised SubQuery Network, the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Agoric from launch.
+
+## Resources
+Here are some additional resources to help you get started:
```
Comment (michaelfig) [inline main/guides/subquery-indexing.md:26]:
```suggestion
Here are some additional resources to help you get started using SubQuery with Agoric:
```

File: main/guides/subquery-indexing.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Running and Hosting your Agoric SubQuery APIs
+SubQuery is open-source, meaning you have the freedom to run it in the following three ways:
+
+* Locally on your own computer, or a cloud provider of your choosing. View the instructions on how to run SubQuery locally [here](https://academy.subquery.network/run_publish/run.html).
+
+* You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/login), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
+
+* You can publish it to the decentralised SubQuery Network, the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Agoric from launch.
```
Comment (michaelfig) [inline main/guides/subquery-indexing.md:23]: Are there any relevant links for the decentralised SubQuery Network?  Again, I'm not sure about making strong claims about this method's superiority in so many ways, unless this guide explicitly quotes the SubQuery team.
```suggestion
* You can publish it to the decentralised SubQuery Network.  The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Agoric from launch.
```

File: main/guides/subquery-indexing.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# SubQuery Indexing
+
+This document explains how to index Agoric blockchain data with SubQuery's open source data indexer.
+
+## Overview
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Agoric Cosmos, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
+
+Features include multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
```
Comment (michaelfig) [inline main/guides/subquery-indexing.md:9]: @sufyaankhan is Agoric willing to back up these claims by putting them in our official documentation, or should they be clarified as being made by the SubQuery team (or be made less strong)?

I would be comfortable with:
```suggestion
As described by the SubQuery team:
> SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Agoric Cosmos, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
>
> Features include multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
```

**Review by michaelfig — COMMENTED** [review]
I want to clarify which parts of this guide are explicitly strong claims made by the SubQuery team, versus which parts are clearly factual and Agoric can represent to the community.

Please take my feedback with a grain of salt, I was just tagged as a reviewer and I have not taken the time to familiarize myself with SubQuery yet.

**Comment by seandotau** [comment]
Thanks @michaelfig for your comments. I've updated the copy based on your suggestions.

File: main/guides/subquery-indexing.md (hunk 4 above)
Comment (sufyaankhan) [reply main/guides/subquery-indexing.md:9]: +1 - @michaelfig I am also comfortable with the language you have suggested.

Comment (seandotau) [reply main/guides/subquery-indexing.md:11]: Thanks @sufyaankhan added the suggestion.

**Review by sufyaankhan — APPROVED** [review]

**Comment by ivanlei** [comment]
closing & reopening to kick stalled GitHub CI.

**Pull request merged**